### DISCUSSION TITLE
GitHubの機能について

### DISCUSSION THREAD

**Discussion by Tooooomo** [KU-eSports/kuest-streaming#52] [Announcements]
### 活用例
- Issueに書く前にDiscussionで大まかなことを話し合っておく
- Discussionでアイデア出しをして、実装が決まったらIssueを立てる
- 分からない点などがあって作業が詰まった時に質問・相談する

***

Issueに関するDiscussionを立てる際は、**"#Issueの番号"** でIssueとリンクを紐づけしてください。
Issue側でも **"#Discussionの番号"** のようにリンクの紐づけをしてください。

### 活用例
- #40
- #52

**Comment by Tooooomo** [comment]
## Projects機能を使ってみましょう

### Projectsとは

↑のリストに並んでいるIssues・Pull requests総覧機能です
とても見やすいうえに現在あるIssuesの閲覧・書き込みもできるのでぜひ活用してみてください
新たにIssue・Pull requestを立てた際にはProjectと紐づけるのを忘れないようにしてください
~~忘れてるのに気づいたら入れておいてね~~

**Reply by secchanu** [reply]
[Projects · kuest-streaming](https://github.com/KU-eSports/kuest-streaming/projects?query=is%3Aopen)

**Comment by Tooooomo** [comment]
## Labels機能を使ってみましょう

### Labelsとは
Issue・Pull request・Discussionの右にある色付きのタグみたいなものです
各項目をタイプや進行度、優先度で分類できます
Issuesのページでソートに使えるほかProjectsでも一覧にしてくれます~~あとデキる人感も出せる~~
Projectsと同じように各項目を立てた際や作業の更新があった時にペタペタ貼り付けてください
もちろんProjectsのページでも貼り替えられますよ！

**Reply by Tooooomo** [reply]
試しにココの作業：立案・ラフのLabelを作業：実装に貼り替えてみてください
IssuesやProjectsに行ってちゃんと並んでるか確認してみてね
- #43 
- #46

**Comment by Tooooomo** [comment]
今気づいたけどここ全部wikiに書いた方がいいね
余裕できたら移行します
